readLog/src/main/java/org/corfudb/readLog/logHeader.java

```diff
@@ -0,0 +1,28 @@
+package org.corfudb.readLog;
+
+import lombok.Getter;
+import lombok.Setter;
+
+/**
+ * Created by kjames88 on 3/1/17.
+ */
+public class logHeader {
```

Comment:
Additional fields vs the protobuf.  Also it shouldn't be necessary to use a protobuf to carry around a few numbers.